src/renderers/dom/shared/DOMProperty.js

```diff
@@ -13,10 +13,26 @@
 
 var invariant = require('fbjs/lib/invariant');
 
+// These attributes should be all lowercase to allow for
+// case insensitive checks
+var RESERVED_PROPS = {
+  children: true,
+  dangerouslysetinnerhtml: true,
+  autofocus: true,
+  defaultvalue: true,
+  defaultchecked: true,
+  innerhtml: true,
+  suppresscontenteditablewarning: true,
+  onfocusin: true,
+  onfocusout: true,
```

Comment:
These two are currently not validated at all. If I type

```js
<div onfocusin={function() {}} />
```

it will silently ignore it. I would expect it to warn.